packages/trpc/server/trpc.ts

```diff
@@ -32,18 +31,15 @@ const isAuthedMiddleware = t.middleware(({ ctx, next }) => {
   });
 });
 
-const isAdminMiddleware = t.middleware(({ ctx, next }) => {
-  if (!ctx.user || !ctx.session || ctx.user.role !== "ADMIN") {
+const isAdminMiddleware = isAuthedMiddleware.unstable_pipe(({ ctx, next }) => {
```

Comment:
we can pipe middlewares so that you don't have to repeat the same logic for multiple middlewares.

Reply:
marked unstable cause api could change in the future but very well tested

Reply:
This is great!